File: ms-comp-inventario/src/main/java/com/uees/mgra/inventario/modals/dto/InsumoDTO.java
package com.uees.mgra.inventario.modals.dto;

import java.time.LocalDate;

public record InsumoDTO(Long id,
                        int cantidad,
                        LocalDate fechaFabricacion,
                        LocalDate fechaExpiracion,
                        boolean esCaducado,
                        Long insumoIdpk){
}

File: front-proyecto/src/app/pages/inventario/interfaces/insumoDTO.ts
export interface InsumoDetalleDTO {
  id: number
  nombre: string
  descripcion: string
  insumoDTOSet: InsumoDTO[]
}

export interface InsumoDTO {
  id: number
  cantidad: number
  fechaFabricacion: string
  fechaExpiracion: string
  esCaducado: boolean
  insumoIdpk: number
}
